userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -195,6 +205,8 @@ Along with price lists, you define <<Rules,catalog rules>> that specify which pr
 
 Subscription Bundle:: See <<Bundle>>
 
+Standalone Product:: A <<product, product>> that cannot have any <<addon,addon>> products. See <<Product_Category, Product Category>>.
```

Comment:
However, we can have have multiple `Standalone` subscriptions within the same `Subscription Bundle` -- as opposed to `Base Product` where we can have only 1 Base subscription (and {0, N} ADDOn subscriptions) within the same bundle.